jp.go

```diff
@@ -130,3 +151,37 @@ func runMain(c *cli.Context) int {
 	os.Stdout.WriteString("\n")
 	return 0
 }
+
+func EnableColor(enabled bool) {
```

Comment:
Still not private :-)

Reply:
I’m strugging to remember to add the `signed-off` to my commits and have to re-do them multiple times 😏.
I will make a quick shortcut to prevent forgetting in the future.

Thanks for bearing with me.